.gitlab-ci.yml

```diff
@@ -39,16 +40,21 @@ cache:
 
 test:runtime:
   stage:                           test
-  # image:                         alpine/git:latest
+  image:                           parity/tools:latest
```

Comment:
What is the difference between these images?

Reply:
`alpine/git` has its ENTRYPOINT set to git, so you can basically only use it for git commands.